pyshelf/bucket_update/search_updater.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, container):
         """
         self.container = container
         self.bucket_container = self.container.bucket_container
-        self.chunk_size = self.container.config["chunkSize"]
+        self.chunk_size = self.container.config.get("chunkSize")
```

Comment:
I don't think this is necessary.  The chunk size shouldn't be required here, its defaulted by the script that invokes it.  Also, if this were to be None stuff would break